examples/camera-controller/webrtc_manager/WebRTCManager.cpp

```diff
@@ -113,29 +120,32 @@ CHIP_ERROR WebRTCManager::Connnect(Controller::DeviceCommissioner & commissioner
         ChipLogProgress(Camera, "[Gathering State: %d]", static_cast<int>(state));
     });
 
-    // Create a data channel for this offerer
-    mDataChannel = mPeerConnection->createDataChannel("test");
-
-    if (mDataChannel)
-    {
-        mDataChannel->onOpen(
-            [&]() { ChipLogProgress(Camera, "[DataChannel open: %s]", mDataChannel ? mDataChannel->label().c_str() : "unknown"); });
-
-        mDataChannel->onClosed([&]() {
-            ChipLogProgress(Camera, "[DataChannel closed: %s]", mDataChannel ? mDataChannel->label().c_str() : "unknown");
-        });
-
-        mDataChannel->onMessage([](auto data) {
-            if (std::holds_alternative<std::string>(data))
-            {
-                ChipLogProgress(Camera, "[Received: %s]", std::get<std::string>(data).c_str());
-            }
-        });
-    }
+    mMedia = rtc::Description::Video("video", rtc::Description::Direction::RecvOnly);
+    mMedia.addH264Codec(96);
```

Comment:
Should not be hardcoded here. How are these values obtained?

Reply:
Controller should be reading the allocated streams on the camera to help plug these values in, no?

Reply:
Thank you for your feedback. We changed it to definition